scripts/globals/spells/spell_enhancing.lua

```diff
@@ -0,0 +1,344 @@
+-----------------------------------
+-- Enhancing Spell Utilities
+-----------------------------------
+require("scripts/globals/spells/parameters")
+require("scripts/globals/spell_data")
+require("scripts/globals/jobpoints")
+require("scripts/globals/status")
+require("scripts/globals/utils")
+require("scripts/globals/msg")
+-----------------------------------
+xi = xi or {}
+xi.spells = xi.spells or {}
+xi.spells.spell_enhancing = xi.spells.spell_enhancing or {}
+-----------------------------------
+-- File structure:
+-- 2 Basic Functions to calculate final potency. Called by the main function.
+-- 1 Basic Function to calculate final duration. Called by the main function.
+-- 1 Main function, called by spell scripts.
+
+-- Table variables.
+local enhancingTable = xi.spells.parameters.enhancingSpell
```

Comment:
I would almost prefer having parameters used in this script here for easier reference instead of one massive file containing params for all different spell types.